lib/tilt/livescript.rb

```diff
@@ -0,0 +1,36 @@
+require 'tilt/template'
+require 'livescript'
+
+module Tilt
+  # LiveScript template implementation. See:
+  # http://livescript.net/
+  #
+  # LiveScript templates do not support object scopes, locals, or yield.
+  class LiveScriptTemplate < Template
+    self.default_mime_type = 'application/javascript'
+
+    @@default_bare = false
+
+    def self.default_bare
+      @@default_bare
+    end
+
+    def self.default_bare=(value)
+      @@default_bare = value
+    end
+
```

Comment:
This is a feature I don't think is very useful. Libraries that integrate with Tilt should give you a way to configure `options` as you'd like. Also, the LiveScript gem itself has a way to set default options globally.